_topic_maps/_topic_map.yml

```diff
@@ -1211,6 +1211,8 @@ Topics:
     File: cert-manager-authenticate
   - Name: Configuring an ACME issuer
     File: cert-manager-operator-issuer-acme
+  - Name: Integrating the cert-manager Operator with Istio-CSR
+    File: cert-manager-operator-integrating-istio
```

Comment:
@swghosh Do you have additional preference for where this page should be placed?

Reply:
Anywhere after "Configuring an Issuer" and before "Uninstallation" page generally works for me. I've no preference as such.

Reply:
I would suggest placing this page right after "Securing routes ...", so that the docs roughly follow a progression from basic to advanced scenarios from top to bottom. Does it make sense?

Please enjoy your day-off @swghosh @subhtk by the way ^^